annotations/fenix/metadata.yaml

```diff
@@ -81,6 +81,8 @@ tags:
     label on GitHub.
   ReaderMode: Corresponds to the [Feature:ReaderMode](https://github.com/mozilla-mobile/fenix/issues?q=label%3AFeature%3AReaderMode)
     label on GitHub.
+  RecentTabs: Corresponds to the [Feature:RecentTabs](https://github.com/mozilla-mobile/fenix/issues?q=label%3AFeature%#3ARecentTabs)
+    label on GitHub.
```

Comment:
Question: If the feature does not have its own label, what should this link point to?

Reply:
Can you add a label for this feature?

Reply:
@amedyne can we add Feature:RecentTabs label for this feature in Fenix repo?

Reply:
@mcarare @wlach I have added the label Feature:RecentTabs to the Fenix repo.

Reply:
Perfect, thank you!